innopoints/views/project.py

```diff
@@ -134,6 +135,13 @@ def publish_project(project_id):
     else:
         abort(401)
 
+    if isinstance(project.moderators, list):
```

Comment:
What else could it possibly be?

Reply:
Good question. I remember while testing it had something else (maybe `None`), but I cannot reproduce this, so I guess I mixed something up